docs/conf.py

```diff
@@ -59,13 +59,27 @@
     'https://doi.org/10.7490/f1000research.1118245.1',
     'https://doi.org/10.7490/f1000research.1118300.1',
     'launch-with/anvil-launch-with.html', 'launch-with/cavatica-launch-with.html', 'launch-with/dnanexus-launch-with.html', 'launch-with/cgc-launch-with.html',
-    'launch-with/galaxy-launch-with.html', 'launch-with/nextflow-tower-launch-with.html', 'launch-with/terra-launch-with.html',
+    'launch-with/galaxy-launch-with.html', 'launch-with/seqera-platform-launch-with.html', 'launch-with/terra-launch-with.html',
```

Comment:
Does it make sense to add to ignore when fixing it in this PR?

Reply:
I think this is some internal issue with the sphinx version/link checker/how we are using it. Notice it's an internal link and has excepted all of the items in the index.rst that link to launch with. 

https://stackoverflow.com/questions/32340062/sphinx-restructuredtext-how-to-link-an-image-to-a-document-page